### PR TITLE
Ergodox EZ refactor

### PR DESCRIPTION
**Pull request by noroadsleft**
Here's me attempting to figure out why the Ergodox layouts in the Configurator don't render properly.

I've had a theory about the API not liking comments in the matrix macros for a while, so this is me testing that.

### CODE REVIEW

File: users/ericgebhart/ericgebhart.h [hunk 1]
```diff
@@ -250,8 +250,8 @@ void x_reset (qk_tap_dance_state_t *state, void *user_data);
 // Since our quirky block definitions are basically a list of comma separated
 // arguments, we need a wrapper in order for these definitions to be
 // expanded before being used as arguments to the LAYOUT_xxx macro.
-#if (!defined(LAYOUT) && defined(KEYMAP))
-#define LAYOUT KEYMAP
```
Comment (drashna) [inline users/ericgebhart/ericgebhart.h]: This shouldn't be changed.

**Comment by drashna** [comment]
Also, you may want to change `layouts/community/ergodox/berfarah/keymap.c`

File: users/ericgebhart/ericgebhart.h (hunk 1 above)
Comment (noroadsleft) [reply users/ericgebhart/ericgebhart.h]: It doesn't compile, at least not for me, without changing it.

With these changes, there is no `KEYMAP` macro. There's only `LAYOUT_ergodox`, `LAYOUT_ergodox_80`, and the `pretty` versions of those to choose from. (I need to make another commit to delete the alias lines in `ergodox_ez.h`.)

If you have a better way of achieving the desired result, I'm all ears.

**Comment by noroadsleft** [comment]
`layouts/community/ergodox/berfarah/keymap.c` is edited in the first commit (the one without the Travis log).

**Comment by drashna** [comment]
Ah, sorry, missed it. 

But the userspace one shouldn't be changed.  It's "correct" for what it's doing.

File: keyboards/ergodox_ez/keymaps/ericgebhart/keymap.c [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
-  [DVORAK] = LAYOUT_wrapper (
+  [DVORAK] = LAYOUT_wrapper(
```
Comment (drashna) [inline keyboards/ergodox_ez/keymaps/ericgebhart/keymap.c]: Should be `LAYOUT_ergodox_wrapper`, if anything.

File: users/ericgebhart/ericgebhart.h (hunk 1 above)
Comment (drashna) [reply users/ericgebhart/ericgebhart.h]: Yeah, because the keymap.c should have `LAYOUT_ergodox_wrapper` rather than `LAYOUT_wrapper`

**Comment by drashna** [comment]
Awesome, thanks

**Pull request merged**